misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -92,3 +94,5 @@ def run(
             *args,
             input=input,
         )
+
+        print(output)
```

Comment:
This likely prints just an empty string, since `capture_output=False` by default.

Reply:
I will remove this since it will already get printed to sysout